src/server/JasmineGraphInstanceService.cpp

```diff
@@ -3318,6 +3475,106 @@ int collectBloomFilters(instanceservicesessionargs *sessionargs,
     return 0;
 }
 
+int JasmineGraphInstanceService::collectBucketstoCoordinator(instanceservicesessionargs *sessionargs,
```

Comment:
Change `collectBucketstoCoordinator` to `collectBucketsToCoordinator`

Reply:
Name change done

Reply:
Ok